app/controllers/CategorisationPreparationController.scala

```diff
@@ -143,7 +145,7 @@ class CategorisationPreparationController @Inject() (
   )(implicit
     hc: HeaderCarrier
   ): Future[Done] =
-    if (categorisationInfo.categoryAssessmentsThatNeedAnswers.isEmpty) {
+    if (categorisationInfo.categoryAssessmentsThatNeedAnswers.isEmpty && !categorisationInfo.isCommCodeExpired) {
```

Comment:
Can test the scenario where commodity code is expired and the categoryAssessmentsThatNeedAnswers is empty to ensure it doesn't go into this case

Reply:
ah right, had it in mind but missed to add, thanks for spotting @ChrisCookOC :)

Reply:
sorted, please check now @ChrisCookOC  :)